### PR TITLE
Prepare PostProcessingPlugin for security.

### PR DESCRIPTION
**Pull request by rburema**

### CODE REVIEW

**Comment by nallath** [comment]
We should only check the scripts in the user folder for signatures (and then of course only if you are in the Enterprise mode). RIght now the check is a bit easy to defeat by simply not having a signature at all :-P

**Comment by Ghostkeeper** [comment]
This PR has no Jira ticket attached. Did this get forgotten? Was it made outside of a Jira ticket?

**Comment by nallath** [comment]
I think there never was a ticket for this. Anwyay, this is something that should be in 4.6, so i've created it (CURA-7319)

**Comment by rburema** [comment]
@Ghostkeeper No I did this during one of the testing and tooling sessions after @nallath talked about how not all cases where covered, even if all the tickets we had at the time where implemented. 

I remember not merging at the time was a conscious decision, but since we didn't close it I assume it's just unfinished in some way (whether that's the PR itself or some other thing that we needed in order to progress I don't know, I'd have to dive in again).

**Comment by rburema** [comment]
I'm not sure why I didn't finish it, all I needed to do was add check to not check the bundled ones.

File: plugins/PostProcessingPlugin/tests/TestPostProcessingPlugin.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    mocked_trust.signedFileCheck = MagicMock(return_value=True)
+
+    realSignatureFileExistsFor = Trust.signatureFileExistsFor
+    Trust.signatureFileExistsFor = MagicMock(return_value=True)
+
+    with patch("UM.Trust.Trust.getInstanceOrNone", return_value=mocked_trust):
+        assert PostProcessingPlugin._isScriptAllowed("blaat.py")
+
+    # cleanup
+    Trust.signatureFileExistsFor = realSignatureFileExistsFor
```
Comment (nallath) [inline plugins/PostProcessingPlugin/tests/TestPostProcessingPlugin.py]: It's a bit cleaner to do this with a patch.object. I'll fix that for you

**Pull request merged**